web/src/pages/starter-kits/general.tsx

```diff
@@ -0,0 +1,160 @@
+              title={Config.starterKits.solutions.card2.title}
+              description={Config.starterKits.solutions.card2.body}
+            />
+            <Card
+              iconFilename="id-badge-circle-icon-green"
+              title={Config.starterKits.solutions.card3.title}
+              description={Config.starterKits.solutions.card3.body}
+            />
+          </div>
+          <div className="cta-button">
+            <PrimaryButton
+              isColor="primary"
+              onClick={() => {
+                analytics.event.starter_kit_landing.click.get_my_starter_kit_button();
+                router.push(ROUTES.onboarding);
+              }}
+              isRightMarginRemoved={true}
+              isLargeButton={true}
+            >
+              {Config.starterKits.solutions.ctaButton}
+            </PrimaryButton>
+          </div>
+        </section>
+        <section
+          aria-label="Example Roadmap"
+          className="display-flex flex-column gap-4 margin-x-2 desktop:margin-x-neg-4 bg-cool-extra-light radius-lg margin-bottom-3 padding-x-2 padding-y-5 desktop:padding-y-10 desktop:padding-x-5"
+        >
+          <div className="padding-x-2 display-flex flex-column flex-justify gap-5 flex-align-center">
+            <div className="border-top-1 border-secondary-vivid-dark width-8" />
+            <Heading level={2}>Get Your Business Started in 4 Easy Steps</Heading>
+          </div>
+          <div className="display-flex flex-column desktop:flex-row gap-4 padding-x-2 desktop:padding-x-0">
+            {props.roadmap.steps.map((step) => (
+              <StepInfo
+                step={step}
+                taskNames={getTaskNamesForStep(props.roadmap.tasks, step.stepNumber)}
+                key={step.stepNumber}
+              />
+            ))}
+          </div>
+          <div className="flex-align-self-stretch desktop:flex-align-self-center tablet:flex-align-self-center">
+            <PrimaryButton
+              isColor={"secondary-vivid-dark"}
+              onClick={() => {
+                analytics.event.starter_kit_landing.click.get_my_starter_kit_button_footer();
+                router.push(ROUTES.onboarding);
+              }}
+              isRightMarginRemoved={true}
+              isLargeButton={true}
+            >
+              {Config.starterKits.steps.ctaButton}
+            </PrimaryButton>
+          </div>
+        </section>
+      </main>
+    </PageSkeleton>
+  );
+};
+
+export const getStaticProps = async (): Promise<{
+  props: {
+    noAuth: boolean;
+    roadmap: Roadmap;
+  };
+}> => {
+  const emptyProfileData = createEmptyProfileData();
+  const newProfileData: ProfileData = {
+    ...emptyProfileData,
+    businessPersona: "STARTING",
+    industryId: "generic",
+  };
+
+  // Next does not accept explicitly undefined values and roadmap has some by default
+  const roadmap = JSON.parse(JSON.stringify(await buildUserRoadmap(newProfileData))) as Roadmap;
+
```

Comment:
It seems like the explicitly undefined culprit here is the `required` field on the `Task` object. I'll have to look into whether we need `required` to be explicitly undefined and if not, convert it to just a boolean. I'll probably be fighting [Hyrum's Law](https://www.hyrumslaw.com/) a bit so we'll see how it goes.

Reply:
Turns out we had some inaccurate typing during the conversion from the roadmap json files to the `AddOn` type which resulted in that undefined field getting explicitly assigned. Commit with explanation to follow.